### ISSUE TITLE
Icon for adding buttons to pages is confusing

### ISSUE THREAD

**Issue by ryanwarsaw** [mozilla/webmaker-core#476]
When you want to add an element to a project in your page, you click the + button and you see two easy to understand buttons, one for text and the other for photo's. The third one looks like a share button, or for a URL redirect, instead it gives you a button. This needs to be made more clear so you know exactly what it does, the instant you open the menu.

**Comment by Pomax** [comment]
I thought we had an issue for this, but we don't!

**Comment by xmatthewx** [comment]
:disappointed: we explored other imperfect options before adding the task to a consulting gig that never happened. Many art apps label their icons to remedy issues like this. 

@LauraReynal - have you seen confusion in the field? Uncomfortable confusion? Or do people try it, learn it, and move on?

**Comment by vazquez** [comment]
We had talked about using this icon instead:

![screenshot 2015-08-12 10 46 04](https://cloud.githubusercontent.com/assets/1119821/9227025/65850036-40df-11e5-895c-0f9cd2e208e3.png)

**Comment by ryanwarsaw** [comment]
Hm, that would be a good icon for creating text links, because that is what I associated that with when I looked at it, (external links), but I don't know if this would be the best choice for a button.

**Comment by Pomax** [comment]
yeah that works for links, but our button is not a link, it's a nav element, so that icon doesn't quite make sense to me

**Comment by lauradereynal** [comment]
The initial button reminded people of "sharing" or other stuff
page 5 here https://docs.google.com/document/d/1EjcWaSi8J6tTeUTB1mSAa8y8xj8RK1YBzCWZL9f_FR0/edit#heading=h.e5g93637x4xz

**Comment by lauradereynal** [comment]
I'm happy to test @vazquez button next time there is an interview
